packages/hatchify-core/src/util/uuidv4.ts

```diff
@@ -1,9 +1,15 @@
-import { getRandomValues } from "crypto"
+function getCrypto() {
```

Comment:
i tried using `can-global` but it did not work as `crypto` is not available as a global in node, it has to be imported. i did copy over the conditional that `can-global` is using under the hood though!